packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -9,6 +9,9 @@ import {
 } from '../../hooks/arbTokenBridge.types'
 import { Tooltip } from './Tooltip'
 
+// Buffer for after a node is confirmable but isn't yet confirmed; we give ~30 minutes, should be usually/always be less in practice
```

Comment:
Better to use a multi-line comment now that it's an exported constant, so the IDE shows description nicely:

```suggestion
/**
 * Buffer for after a node is confirmable but isn't yet confirmed; we give ~30 minutes, should be usually/always be less in practice.
 */
```

Reply:
done